apps/Steam/install-64

```diff
@@ -32,7 +32,7 @@ fi
 "${DIRECTORY}/manage" install-if-not-installed Box64 || error "Box64 failed to install somehow!"
 
 echo "Installing steam.deb"
-install_package https://steamcdn-a.akamaihd.net/client/installer/steam.deb || error "Could not install steam deb"
+install_packages https://steamcdn-a.akamaihd.net/client/installer/steam.deb || error "Could not install steam deb"
```

Comment:
shouldn't this be merged into the install_packages used above @ https://github.com/Botspot/pi-apps/pull/653/commits/e44c16694666335259f199ff4f4efca7d91c22bd#diff-9dacdd03ffbd2756a2e27273b45e074f8695ae87b835a18d5e6cf00297463d0dR17

Reply:
Then its better to install all the steam libs here

Reply:
I would rather keep the armhf dependencies and steam as separate